### DISCUSSION TITLE
checking before i install

### DISCUSSION THREAD

**Discussion by quakeman00** [TagStudioDev/TagStudio#313] [Q&A]
i am looking to know if it support ANY files

i have very old files and even flash files, sfw, and sprites files in bastard format nobody uses anymore and only work with that game, but i would like to tag em anyway, does this support anything thats in a folder?

**Comment by samuellieberman** [comment]
> i am looking to know if it support ANY files
> 
> i have very old files and even flash files, sfw, and sprites files in bastard format nobody uses anymore and only work with that game, but i would like to tag em anyway, does this support anything thats in a folder?

![image](https://github.com/TagStudioDev/TagStudio/assets/51455162/cc1e9b63-0617-4769-b1f7-f356671e4dbc)
Looks all good to me!

**Reply by quakeman00** [reply]
Amazing, i will bookmark this and download it when the dev itself feel like the program is in an acceptable usable state, the github page is full of "EVERYTHING'S GONNA BREAK" right now
